src/librustc/mir/visit.rs

```diff
@@ -247,42 +247,30 @@ macro_rules! make_mir_visitor {
 
             fn super_mir(&mut self,
                          mir: & $($mutability)* Mir<'tcx>) {
-                let Mir {
```

Comment:
what is the motivation for making this private? I am unhappy about this change, because now when new things are added to mir, we no longer get errors in the visitor.